### PR TITLE
change ID in community.html to "userlist"

### PR DESCRIPTION
**Pull request by Ivyzhang1998**
for some reasons it does not load data in community.html. I double checked with the original code and fixed it by adding two whitespace. Now it works.

### CODE REVIEW

File: src/main/webapp/community.html [hunk 1]
```diff
@@ -16,8 +16,8 @@
           userlist.innerHTML = '';
 
           users.forEach((user) => {
-          const userListItem = buildUserListItem(user);
```
Comment (NickChapman) [inline src/main/webapp/community.html:19]: Hmmmmm, this seems highly suspect to me. Do you have any more details on what the error is without these white spaces?

File: src/main/webapp/community.html [hunk 2]
```diff
@@ -12,9 +12,8 @@
         fetch(url).then((response) => {
           return response.json();
         }).then((users) => {
-          const userlist = document.getElementById('userlist');
+          const userlist = document.getElementById('list');
```
Comment (NickChapman) [inline src/main/webapp/community.html]: I would go the other way and err on the side of more details. So I would update the element `id` to be `userlist` and leave the javascript part as it is.

**Review by NickChapman — CHANGES_REQUESTED** [review]
Quick comment, also don't forget to update the PR description :slightly_smiling_face:

File: src/main/webapp/community.html (hunk 2 above)
Comment (Ivyzhang1998) [reply src/main/webapp/community.html]: resolved!

**Review by NickChapman — APPROVED** [review]

**Pull request merged**